woocommerce/woocommerce
# Add order support to API package

### All Submissions:


### Changes proposed in this Pull Request:

This PR completes work on adding order support to the API Package. Additionally, it adds a test around that work and removes an unused import from an existing e2e tests.

Closes https://github.com/woocommerce/woocommerce/issues/27873

### How to test the changes in this Pull Request:

1. Verify the CI runs and passes
2. Run the new test for the API changes and verify it passes `npx wc-e2e test:e2e tests/e2e/specs/rest-api/order.js`
3. Verify the e2e test suite passes

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
